pkg/status/handlers.go

```diff
@@ -37,83 +34,46 @@ type API struct {
 func (a *API) RegisterEndpoints(mux *chi.Mux) {
 	mux.Get("/api/v0/status", a.alive)
 	mux.Get("/api/v0/version", a.version)
-	mux.Get("/api/v0/deepcheck", a.deepCheck)
+	mux.Get("/api/v0/health", a.health)
```

Comment:
nitpick: I would prefer `ready` instead `health` as endpoint name. This is a readiness endpoint (checks the application's state as well as the dependencies' state). But since we don't have a liveness endpoint, I have no problem with keeping this name.